packages/itwinui-react/src/core/Buttons/IdeasButton.test.tsx

```diff
@@ -7,16 +7,24 @@ import * as React from 'react';
 
 import { IdeasButton } from './IdeasButton.js';
 
+const assertBaseElement = (container: HTMLElement) => {
```

Comment:
nit: no need for unnecessary abstractions inside tests.

Reply:
Since I found identical code, I thought to abstract them into `assertBaseElement()`, similar to other unit tests. So, not sure of the advantage of not abstracting common code in this case.

But I don't mind removing it too since the unit test file is anyways small. Reverted the abstraction (https://github.com/iTwin/iTwinUI/pull/1919/commits/a8e0b684479d9e89eb34e14cac0ef602fb88e3d0).

Reply:
@rohan-kadkol The unit tests are failing.

Reply:
Ohh, fixed (https://github.com/iTwin/iTwinUI/pull/1919/files/55a8896a10abf201f1eae790b8552b845aa168a6..436c5404f92a06216cef4188801c3cf86fcf1cdf)